schema/partner_shows.js

```diff
@@ -0,0 +1,40 @@
+import gravity from '../lib/loaders/gravity';
+import PartnerShow from './partner_show';
+import {
+  GraphQLString,
+  GraphQLList,
+  GraphQLInt,
+  GraphQLBoolean
+} from 'graphql'
+
+let PartnerShows = {
+  type: new GraphQLList(PartnerShow.type),
+  description: 'A list of PartnerShows',
+  args: {
+    size: {
+      type: GraphQLInt
+    },
+    sort: {
+      type: GraphQLString
```

Comment:
Example of how this should be typed: https://github.com/artsy/metaphysics/blob/103b3ccb39c801bad0abbd65238c3375e90d9eab/schema/artists.js#L16-L24